### DISCUSSION TITLE
Runtime error

### DISCUSSION THREAD

**Discussion by CA10C3SAR** [zero-plusplus/vscode-autohotkey-debug#100] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/6724006/115103512-c787b500-9f28-11eb-94a1-d1e076dae759.png)

When i click open launch.json it pops this box:
![image](https://user-images.githubusercontent.com/6724006/115103532-e4bc8380-9f28-11eb-914a-60eceaae5f6e.png)

Looks like i dont have this file.
Where i can get it and where should it be placed?

**Comment by zero-plusplus** [comment, marked as answer]
If you install AutoHotkey, you can start debugging without worrying about launch.json.

The main cases where you will need launch.json are if you are using a portable version of AutoHotkey that can be used without installation, or if you want to use advanced debugger features.

If you want to learn about launch.json itself, see [here](https://code.visualstudio.com/docs/editor/debugging#_launch-configurations).
